dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm

```diff
@@ -4060,16 +4060,16 @@ sub generate_output {
 
     # Emit mortalisation and set magic code on the result SV if need be
 
-    push @lines, "\tRETVAL".($use_RETVALSV ? 'SV':'')
-          .' = sv_2mortal(RETVAL'.($use_RETVALSV ? 'SV':'').");\n" if $do_mortal;
-    push @lines, "\tSvSETMAGIC(RETVAL".($use_RETVALSV ? 'SV':'').");\n" if $do_setmagic;
+    my $sv = $use_RETVALSV ? 'SV' : '';
```

Comment:
This is an improvement, but I think it would be even more readable if it were something like:

`my $retval = $use_RETVALSV ? 'RETVALSV' : 'RETVAL'`

So basically a variable that contains the whole identifier, rather than just a suffix or empty string.

Reply:
Thanks, I'll address that in a future commit.